assets/js/phoenix_live_view/live_socket.js

```diff
@@ -400,7 +400,7 @@ export default class LiveSocket {
 
     this.main = this.newRootView(newMainEl, flash, liveReferer)
     this.main.setRedirect(href)
-    this.transitionRemoves()
+    this.transitionRemoves(null, true)
```

Comment:
Any chance we can make this the default behavior? eg. reversing the parameter sense: `includingStickies[=false]`